database/nosql/cassandra/egg-apache-cassandra.json

```diff
@@ -0,0 +1,60 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-08-03T22:11:40+02:00",
+    "name": "Apache Cassandra",
+    "author": "p.zarrad@outlook.de",
+    "description": "Apache Cassandra is an open source NoSQL distributed database trusted by thousands of companies for scalability and high availability without compromising performance. Linear scalability and proven fault-tolerance on commodity hardware or cloud infrastructure make it the perfect platform for mission-critical data.",
+    "features": null,
+    "images": [
```

Comment:
Can you PR these images to the yolks repo?

Reply:
Don't have time right now. I will do it in the next two weeks.
I will also have to re-check the egg and fix conflicsts.

Reply:
I checked my repo and there is a cassandra yolk no idea what I was looking at that didn't show it.